samples/test/steps/then.ts

```diff
@@ -5,15 +5,16 @@ import checkIsOnPage from '../support/check/checkIsOnPage';
 import checkProfile from '../support/check/checkProfile';
 import checkFormMessage from '../support/check/checkFormMessage';
 import checkGuest from '../support/check/checkGuest';
+import checkURLPath from '../support/check/checkURLPath';
 
 Then(
   /^User can verify their profile data$/,
-  checkProfile
+  () => checkProfile()
```

Comment:
do we need this? `checkProfile` is already a function.

Reply:
cucumber gets very upset if the number of parameters don't match. `checkProfile` has an optional parameter (`falseCase`). However, these functions should also be async. So maybe something like:

```
async () => await checkProfile()
```

But it may actually be better to remove the optional parameter and replace with two functions. Something like:`assertProfile` and `assertNoProfile`